app/src/main/java/net/gsantner/opoc/ui/SearchOrCustomTextDialog.java

```diff
@@ -136,15 +144,17 @@ public View getView(int pos, @Nullable View convertView, @NonNull ViewGroup pare
 
             final TextView textView;
             if (convertView == null) {
-                textView = (TextView) _inflater.inflate(_dopt.isMultiSelectEnabled ? android.R.layout.simple_list_item_multiple_choice : android.R.layout.simple_list_item_1, parent, false);
+                textView = (TextView) _inflater.inflate(_layout, parent, false);
+                ViewGroup.LayoutParams params = textView.getLayoutParams();
+                params.height = ViewGroup.LayoutParams.WRAP_CONTENT;
+                textView.setMinHeight(_layoutHeight);
             } else {
                 textView = (TextView) convertView;
             }
 
             if (textView instanceof Checkable) {
                 ((Checkable) textView).setChecked(_selectedItems.contains(index));
             }
-            textView.setLayoutParams(new LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.WRAP_CONTENT));
```

Comment:
Layout setting moved into if - don't need to set layout every time. Just on creation.